lms/templates/fields/field_dropdown_account.underscore

```diff
@@ -20,6 +20,14 @@
         <span class="u-field-value-readonly"></span>
     <% } else { %>
         <select name="select" id="u-field-select-<%- id %>" aria-describedby="u-field-help-message-<%- id %>">
+            <% if (preOptions) { %>
+                <optgroup label="Common Time Zone">
+                <% _.each(preOptions, function(time_zone) { %>
+                   <option value="<%- time_zone.time_zone %>"><%- time_zone.description %></option>
+                <% }); %>
+                <option disabled>──────────</option>
```

Comment:
@kevinjkim I'd look at how the LMS's main drop down achieves this effect. You can see it on the top right:

https://stage.edx.org/

You can also see it in the Pattern Library doc:

http://ux.edx.org/components/dropdown-menu/